src/CSET/operators/plot.py

```diff
@@ -568,7 +571,16 @@ def _plot_and_save_scatter_plot(
         Whether a 1:1 line is plotted.
     """
     fig = plt.figure(figsize=(10, 10), facecolor="w", edgecolor="k")
-    iplt.scatter(cube_x, cube_y)
+    # plot the cube_x and cube_y 1D fields as a scatter plot. If they are CubeLists this ensures
+    # to pair each cube from cube_x with the corresponding cube from cube_y, allowing to iterate
+    # over the pairs simultaneously.
+
+    # Ensure cube_x and cube_y are iterable turning them into CubeLists
+    cube_x_iterable = iter_maybe(cube_x)
+    cube_y_iterable = iter_maybe(cube_y)
```

Comment:
```suggestion
    # Ensure cube_x and cube_y are iterable.
    cube_x_iterable = iter_maybe(cube_x)
    cube_y_iterable = iter_maybe(cube_y)
```
Strictly speaking they are just lists, not CubeLists.

Reply:
done